src/stores/events/EventsStore.ts

```diff
@@ -181,41 +185,115 @@ export default class EventsStore {
 		}
 	};
 
-	private eventTreeAC: AbortController | null = null;
-
 	@action
-	private fetchEventTree = async () => {
-		if (this.eventTreeAC) {
-			this.eventTreeAC.abort();
-		}
-		this.eventTreeAC = new AbortController();
-		this.eventTreeStatusCode = null;
+	private fetchEventTree = () => {
```

Comment:
store is getting quite large, maybe we can create some EventsDataStore to move everything related to fetching there?